stubs/backend/buildkite/buildkite-stubs.yaml

```diff
@@ -26,14 +26,14 @@
 # Pipeline builds
 - request:
     method: GET
-    url: /v2/organizations/XXXX/pipelines/(.*)/builds
+    url: /v2/organizations/XXXX/pipelines/(fs-[a-zA-Z-]+)/builds
     query:
-      page: 1
+      page: "(\\d+)"
```

Comment:
this pr has too many file changes, is there something wrong?

Reply:
I have rebase the brach again, now it's clearly.